python/delta/tests/test_deltatable.py

```diff
@@ -1188,16 +1188,19 @@ def test_protocolUpgrade(self) -> None:
         with self.assertRaisesRegex(ValueError, "writerVersion"):
             dt.upgradeTableProtocol(1, {})  # type: ignore[arg-type]
 
-    def test_addFeatureSupport(self) -> None:
+    def __create_df_for_feature_tests(self) -> DeltaTable:
```

Comment:
Should we also use this refactor `__create_df_for_feature_tests` in other place like `test_protocolUpgrade`?